src/v/kafka/protocol/fetch.h

```diff
@@ -489,6 +489,10 @@ class partition_wrapper {
                     : _partition->high_watermark();
     }
 
+    model::offset start_offset() const {
+        return _log ? _log->offsets().start_offset : _partition->start_offset();
```

Comment:
@mmaslankaprv can you make a comment on the difference of these 2.

Reply:
you are right, we can always access `partition->start_offset`.